Dockerfile

```
@@ -0,0 +1,19 @@
+FROM ruby:3.0.3
+
+RUN apt-get update -qq && apt-get install -y nodejs postgresql-client && apt-get install -y --no-install-recommends yarn
```

I think you can use the `apt-get` passing more than one package, something like this:

```suggestion
RUN apt-get update -qq && apt-get install -y --no-install-recommends nodejs postgresql-client yarn
```